### PR TITLE
Bump version to 2.0.3

### PR DESCRIPTION
**Pull request by tblivet**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Bump version to 2.0.3
| Type?             |  new feature
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | -
| Sponsor company   | @PrestaShopCorp
| How to test?      | -

### CODE REVIEW

**Review by jolelievre — APPROVED** [review]

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
